<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWin</class>
 <widget class="QMainWindow" name="MainWin">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWin</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="B0_Canny">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>540</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Canny/坎尼</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B1_Sobel">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>540</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Sobel/索贝尔</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B2_Kirsch">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>540</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Kirsch</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B3_Laplacian">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>540</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Laplacian/拉普拉斯</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B4_SUSAN">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>600</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>SUSAN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B7_save">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>600</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>保存图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B6_resset">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>600</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>还原图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B5_Harris">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>600</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Harris/哈里斯</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="S1_brigthness">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>550</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="sliderPosition">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="S2_contrace">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>590</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="sliderPosition">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="P_1">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>30</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>图片一</string>
    </property>
   </widget>
   <widget class="QPushButton" name="P_2">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>90</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>图片二</string>
    </property>
   </widget>
   <widget class="QPushButton" name="P_3">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>150</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>图片三</string>
    </property>
   </widget>
   <widget class="QPushButton" name="P_4">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>210</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>图片四</string>
    </property>
   </widget>
   <widget class="QPushButton" name="P_5">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>270</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>图片五</string>
    </property>
   </widget>
   <widget class="QPushButton" name="P_6">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>330</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>图片六</string>
    </property>
   </widget>
   <widget class="QPushButton" name="P_7">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>390</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PIC-7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="P_8">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>450</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PIC-8</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>550</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>亮度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>590</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>对比度</string>
    </property>
   </widget>
   <widget class="QLabel" name="S1_value">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>550</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
   </widget>
   <widget class="QLabel" name="S2_value">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>590</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>630</y>
      <width>48</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>630</y>
      <width>48</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>630</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>MIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>630</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>MAX</string>
    </property>
   </widget>
   <widget class="QLabel" name="ori_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>601</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 87, 83);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="debug_win">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>60</y>
      <width>451</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>当前图片</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>debug窗口</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumain">
    <property name="title">
     <string>main</string>
    </property>
   </widget>
   <addaction name="menumain"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
